app/src/main/java/org/projectbuendia/client/models/ObsRow.java

```diff
@@ -15,15 +15,19 @@
     public String day;
     public String time;
     public String conceptName;
+    public String conceptUid;
```

Comment:
This should be spelled `conceptUuid`—could you rename both the field and the constructor argument?

Reply:
Done